torch/_refs/__init__.py

```diff
@@ -944,47 +949,70 @@ def add(
 
 
 # TODO: add docstring
-atan2 = _make_elementwise_binary_reference(
-    prims.atan2,  # type: ignore[has-type]
+@partial(
```

Comment:
I don't love the pattern of `@partial` for this and then having to basically redefine the prim in a wrapper. Are there some other options?

Reply:
I'll remove all the partials in this PR as well.